jobspec2/parse_test.go

```diff
@@ -94,6 +193,52 @@ job "example" {
 
 	require.Equal(t, meta, out.Meta)
 }
+func TestParse_Locals(t *testing.T) {
+	hcl := `
+variables {
+  region_var = "default_region"
+}
+
+locals {
+  # literal local
+  dc = "local_dc"
+  # local that depends on a variable
+  region = "${var.region_var}.local"
```

Comment:
This is totally a nitpick, but I found this test input really confusing to read because it's using the literal `.local` in the value here, which doesn't have anything to do with the `local` variables and just happens to collide textually.

If this is intentional to verify we don't have weird parsing bugs, let's call that out specifically? Or rename it to `.example` or something like that.